server/pulp/server/webservices/views/permissions.py

```diff
@@ -0,0 +1,189 @@
+from django.views.generic import View
+
+from pulp.server import exceptions as pulp_exceptions
+from pulp.server.auth import authorization
+from pulp.server.managers import factory
+from pulp.server.webservices.controllers.decorators import auth_required
+from pulp.server.webservices.views.util import (generate_json_response,
+                                                generate_json_response_with_pulp_encoder)
+
+
+class PermissionView(View):
+    """
+    Views for permissions retrieval.
+    """
+
+    @auth_required(authorization.READ)
+    def get(self, request):
+        """
+        Retrieve permissions for all resources or for a particular resource.
+
+        :param request: WSGI request object
+        :type request: django.core.handlers.wsgi.WSGIRequest
+        :return: Response containing a list of permissions for resource/s
```

Comment:
I'll suggest an extra line of whitespace between the return params (:return, :rtype) and the argument params for readability. Not required, but a stylistic suggestion.